### ISSUE TITLE
distanceInWordsToNow doesnt work with locales

### ISSUE THREAD

**Issue by stasoft91** [nuxt-community/date-fns-module#1]
```
export default {
  data() {
    return {
      dateFormatted: this.$dateFns.distanceInWords(new Date(), new Date(), {
        locale: 'it'
      }),
      dateFormatted2: this.$dateFns.distanceInWordsToNow(new Date(), {
        locale: 'it'
      })
    }
  }
}
```

{{ dateFormatted }} - meno di un minuto
{{ dateFormatted2 }} - less than a minute ago

**Comment by ricardogobbosouza** [comment]
Hi @stasoft91 

You imported locale?

**Comment by stasoft91** [comment]
Hi @ricardogobbosouza 
Sure, in the same environment distanceInWords works fine, while distanceInWordsToNow not
That is if by importing you mean 
```
modules: [[
      '@nuxtjs/date-fns',
      {
        locales: languages.map(lang => lang.locale),
        defaultLocale: 'ru',
        format: 'YYYY-MM-DD'
      }
    ]]
```

**Comment by ricardogobbosouza** [comment]
Hi @stasoft91
Fixed on version `^0.0.3`
